src/awsexplorer/commands/copyName.ts

```diff
@@ -3,25 +3,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { DefaultEnv, Env } from '../../shared/vscode/env'
-import { DefaultWindow, Window } from '../../shared/vscode/window'
+import { Env } from '../../shared/vscode/env'
+import { Window } from '../../shared/vscode/window'
 import { getLogger } from '../../shared/logger'
 import { AWSResourceNode } from '../../shared/treeview/nodes/awsResourceNode'
 import { localize } from '../../shared/utilities/vsCodeUtils'
 
+const COPY_NAME_DISPLAY_TIMEOUT_MS = 2000
+
 /**
  * Copies the name of the resource represented by the given node.
  */
 export async function copyNameCommand(
     node: AWSResourceNode,
-    window: Window = new DefaultWindow(),
-    env: Env = new DefaultEnv()
+    window = Window.vscode(),
+    env = Env.vscode()
 ): Promise<void> {
-    getLogger().debug(`CopyName called for ${node}`)
+    getLogger().debug('CopyName called for %O', node)
     await env.clipboard.writeText(node.name)
+    getLogger().info(`Copied name ${node.name} to clipboard`)
     recordCopyName()
 
-    window.setStatusBarMessage(localize('AWS.explorerNode.copiedName', 'Copied name to clipboard'))
+    window.setStatusBarMessage(
```

Comment:
Is this expected? For instance, right clicking a file explorer node -> "Copy Path" doesn't pop a message.

Reply:
Not sure what you mean. Copy Path and Copy ARN also show the same status bar message.

Reply:
It's mainly a nit, but I'm talking about VSCode-native, non-AWS actions (e.g. Copy Relative Path on any file in the VS Code file explorer). Does the user need the feedback that the copy was made? You could argue that the context menu disappearing handles this (I usually see "Copied X..." messages if you click a button and the button doesn't disappear or change).

Reply:
I just checked and stuff like `Copy Path` and `Copy Relative Path` on file explorer items show no feedback whatsoever. The status bar messages were actually already there when `copyArn` was added. I actually like this better than no feedback. Verification that you didn't misclick and copy the ARN instead of the name, for example.